tests/integration/takeovers/test_resolve_command.py

```diff
@@ -1499,10 +1499,10 @@ async def test_abandon_controlling_same_block_as_new_claim(self):
         await self.assertNameState(272, name, second_claim_id, last_takeover_height=272, non_winning_claims=[])
 
     async def test_trending(self):
-        async def get_trending_score(claim_id):
+        async def get_claim(claim_id):
             return (await self.conductor.spv_node.server.session_manager.search_index.search(
                 claim_id=claim_id
-            ))[0][0]['trending_score']
+            ))[0][0]
```

Comment:
this change appears to make the tests more verbose, can you remain the way it was before?

Reply:
Reverted...